controllers/managedocs_controller_test.go

```diff
@@ -618,9 +628,8 @@ var _ = Describe("ManagedOCS controller", func() {
 			})
 			When("there is an invalid storage unit value in the add-on parameters secret", func() {
 				It("should not create reconciled resources", func() {
-					if testReconciler.DeploymentType != consumerDeploymentType {
-						Skip(fmt.Sprintf("Skipping the test as it is not required by %v", testReconciler.DeploymentType))
-					}
+					// quota size isn't enforced by provider when consumer supplies it
+					Skip(fmt.Sprintf("Skipping the test as it is not required by %v", testReconciler.DeploymentType))
```

Comment:
Why keep the test if we deprecated the `unit` addon parameter?

Reply:
Ack, I'll remove standalone consumer tests involving size & unit.